cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/EquipmentExport.java

```diff
@@ -241,13 +243,70 @@ private static void writeBusbarSections(Network network, String cimNamespace, XM
         }
     }
 
-    private static void writeLoads(Network network, String cimNamespace, XMLStreamWriter writer, CgmesExportContext context) throws XMLStreamException {
+    static class LoadGroup {
```

Comment:
You can write an external class, it will increase readability.

Reply:
moved to external class